### PR TITLE
whitelist-spacy

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist-spacy

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -1977,6 +1977,900 @@
...
+    "spacy.cli.convert": [],
+    "spacy.cli.debug_config": [],
+    "spacy.cli.debug_data": [],
+    "spacy.cli.debug_diff": [],
+    "spacy.cli.debug_model": [],
+    "spacy.cli.download": [],
+    "spacy.cli.evaluate": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'msg.table([(k, f'{v:.2f}' if isinstance(v, (float, int)) else v) for (k, v) in scores.items()], header=('', 'ROC AUC'), aligns=('l', 'r'), title='Textcat ROC AUC (per label)')' because its table name argument cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:1997]: noise

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -1977,6 +1977,900 @@
...
+      }
+    ],
+    "spacy.cli.find_function": [],
+    "spacy.cli.find_threshold": [],
+    "spacy.cli.info": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'msg.table(table_data, title=title)' because its table name argument cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2005]: this is irrelevant as well

File: src/databricks/labs/ucx/source_code/known.json [hunk 3]
```diff
@@ -1977,6 +1977,900 @@
...
+    "spacy.ml.models.spancat": [],
+    "spacy.ml.models.tagger": [],
+    "spacy.ml.models.textcat": [],
+    "spacy.ml.models.tok2vec": [],
+    "spacy.ml.staticvectors": [],
+    "spacy.ml.tb_framework": [],
+    "spacy.pipe_analysis": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'msg.table(body, header=header, divider=True, multiline=True)' because its table name argument cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2448]: this is not `spark.table`

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: done

File: src/databricks/labs/ucx/source_code/known.json (hunk 3 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: removed, filed https://github.com/databrickslabs/ucx/issues/2017

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: done

**Review by nfx — APPROVED** [review]
lgtm

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**